test/daterangepicker.tests.js

```diff
@@ -1098,7 +1132,7 @@ define([
                     this.options = options;
                 }
                 TestPlugin.pluginName = 'testPlugin';
-                TestPlugin.prototype.attach = sinon.spy();
+                TestPlugin.prototype.attach = sandbox.spy();
```

Comment:
Does `attach` exist on the TestPlugin's `proto`? If yes `sandbox.stub(TestPlugin.prototype, 'attach') would be safer I think. Cause this can actually not be restored from what I know.